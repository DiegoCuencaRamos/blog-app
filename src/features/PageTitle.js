import React from 'react'

const PageTitle = ({ title, description }) => (
    <div className="page-header">
        <div className="container">
            <h1 className="page-header__title">{title}</h1>
            <h5 className="page-header__text">{description}</h5>
            </div>
    </div>
)

export default PageTitle

