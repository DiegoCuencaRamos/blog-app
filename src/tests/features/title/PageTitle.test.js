import React from 'react'
import { shallow } from 'enzyme'
import PageTitle from '../../../features/PageTitle'

test('should render PageTitle correctly', () => {
    const wrapper = shallow(<PageTitle title={'Some title'} />)
    expect(wrapper).toMatchSnapshot()
})