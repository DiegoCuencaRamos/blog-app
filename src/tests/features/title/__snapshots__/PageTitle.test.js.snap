// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PageTitle correctly 1`] = `
<div
  className="page-header"
>
  <div
    className="container"
  >
    <h1
      className="page-header__title"
    >
      Some title
    </h1>
    <h5
      className="page-header__text"
    />
  </div>
</div>
`;
